### ISSUE TITLE
Missing regions of PR

### ISSUE THREAD

**Issue by Klariff** [dr5hn/countries-states-cities-database#565]
When I fetch the get states endpoint for PR as a country, the api only returns me 10 regions of Puerto Rico, there are 78 municipalities

**Comment by dr5hn** [comment]
@Klariff Can you please confirm if the specified municipalities can be found in the cities database?

**Comment by Klariff** [comment]
Yeah I can find them there by doing country US and state PR, but that query also returns "Barrios" from PR which are a different level of territorial administration
